compiler_setup/nvidia.sh

```diff
@@ -0,0 +1,51 @@
+# Build settings for the Nvidia compiler
```

Comment:
I think the two pgi scripts should be merged or removed at some point.

Reply:
I've now removed both the pgi.sh and pgi_acc.sh compiler set up files.